virttest/test_setup.py

```diff
@@ -64,6 +64,14 @@ class HPNotSupportedError(Exception):
     pass
 
 
+class HPSetupError(Exception):
+
+    """
+    Thrown when hugepages setup error.
+    """
+    pass
```

Comment:
As you raise this Exception with detail msg, it's better to implement '__str__()' for this class